cmd/apiserver/app/server/server.go

```diff
@@ -205,6 +211,7 @@ func (serverOptions ServiceCatalogServerOptions) runServer() error {
 
 	// I don't like this. We're reaching in too far to call things.
 	preparedserver := server.GenericAPIServer.PrepareRun() // post api installation setup? We should have set up the api already?
+	preparedserver.SecureServingInfo = nil
```

Comment:
@pmorie I think I added this right when we started working on this PR together. I don't think it's necessary. Thoughts?